docs/books/bash_programming/01-Shell-overview.md

```diff
@@ -35,11 +35,133 @@ Shell classification:
 * Bourne Shell - This family includes but is not limited to:
     * sh (Bourne Shell, /usr/bin/sh). It was developed at Bell LABS in 1977 by Stephen Bourne and used on V7 UNIX
     * ksh (Korn Shell, /usr/bin/ksh)
-    * Bash (GNU Bourne-Again Shell, /bin/bash) - Born in 1987, it is the product of the GNU Project. Most GNU/Linux operating systems use bash as their default shell
-    * psh (POSIX Shell)
-    * zsh (Z-shell)
+    * Bash (GNU Bourne-Again Shell, /bin/bash) - Born in 1987, it is the product of the GNU Project. Most GNU/Linux operating systems use bash as their default shell. For more detailed information, please [read this manual](https://www.gnu.org/software/bash/manual/bash.html).
+    * psh (POSIX Shell) - For more detailed information, please [read this specification](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html).
+    * zsh (Z-shell) - For more detailed information, please [read here](https://zsh.sourceforge.io/).
 * C Shell - This family includes but is not limited to:
     * csh
-    * tcsh
+    * tcsh - For more detailed information, please [visit this website](https://www.tcsh.org/)
 * Power Shell
 
+## Fence Code Block style description
+
+* Translators should translate the annotations on the command.
+
+  ```bash
+  # Lists the files in the current directory
+  bash > ls
+  ```
+
+* If you need to break a line to explain the meaning of a command, you can use two or more "#". Of course, this approach also applies to associated operation steps.
+
+  ```bash
+  # First decompress
+  bash > tar -zvxf nginx-1.21.1.tar.gz -C /usr/local/src/
+  ## And then go into the directory
+  bash > cd /usr/local/src/nginx-1.21.1/
+  ```
+
+* Declare the user and current path.
+  
+  ```bash
+  bash (root /root/)> ls -l
+
+  bash (root)>
+
+  bash (/tmp/)>
+  ```
+
+* Use arrows to illustrate the meaning of command output.
```

Comment:
@jimcat8  - Who are notes like these ^^^ meant for ? 

I could be mistaken but -  if they are meant for translators or editors or other contributors then they don't belong directly in this book/guide/doc.  
And if the notes are meant as a form of reminder to yourself to do something later on - please convert this to work into a draft.